### PR TITLE
Quantified code 2

### PR DESCRIPTION
**Pull request by fernandog**
as title

### CODE REVIEW

File: sickbeard/event_queue.py [hunk 1]
```diff
@@ -35,15 +35,15 @@ def run(self):
             while not self.stop.is_set():
                 try:
                     # get event type
-                    type = self.queue.get(True, 1)
+                    event_type = self.queue.get(True, 1)
 
                     # perform callback if we got a event type
-                    self.callback(type)
+                    self.callback(get_type)
```
Comment (p0psicles) [inline sickbeard/event_queue.py]: that's not right

File: sickbeard/metadata/generic.py [hunk 2]
```diff
@@ -325,7 +325,7 @@ def create_season_posters(self, show_obj):
                 if not self._has_season_poster(show_obj, season):
                     logger.log(u"Metadata provider " + self.name + " creating season posters for " + show_obj.name,
                                logger.DEBUG)
-                    result = result + [self.save_season_posters(show_obj, season)]
+                    result = result.extend(self.save_season_posters(show_obj, season))
```
Comment (labrys) [inline sickbeard/metadata/generic.py]: This is wrong.  either `result +=` or `result.extend`

File: sickbeard/postProcessor.py [hunk 3]
```diff
@@ -1009,7 +1009,7 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
                 self._log(u"File exists and new file is same size, pretending we did something")
                 return True
 
-            if new_ep_quality <= old_ep_quality != common.Quality.UNKNOWN and existing_file_status != PostProcessor.DOESNT_EXIST:
+            if new_ep_quality <= old_ep_quality and old_ep_quality != common.Quality.UNKNOWN and existing_file_status != PostProcessor.DOESNT_EXIST:
```
Comment (labrys) [inline sickbeard/postProcessor.py]: use all() here.

File: sickbeard/event_queue.py [hunk 4]
```diff
@@ -7,11 +7,15 @@
...
 
 class Event(object):
-    def __init__(self, type):
-        self._type = type
+    def __init__(self, get_type):
+        self._type = get_type
 
     @property
-    def type(self):
+    def get_type(self):
```
Comment (labrys) [inline sickbeard/event_queue.py]: `get_type` is a bad name for a property, its how the method should be named

File: sickbeard/postProcessor.py (hunk 3 above)
Comment (fernandog) [inline sickbeard/postProcessor.py]: @labrys didn't understand

File: sickbeard/logger.py [hunk 5]
```diff
@@ -75,13 +75,13 @@ def __init__(self, fmt=None, datefmt=None, encoding='utf-8'):
         super(CensoredFormatter, self).__init__(fmt, datefmt)
         self.encoding = encoding
 
-    def format(self, record):
+    def censor_strings(self, record):
```
Comment (labrys) [inline sickbeard/logger.py]: this would be more properly named `censor_message`

File: sickbeard/event_queue.py (hunk 4 above)
Comment (fernandog) [inline sickbeard/event_queue.py]: @labrys also didn't understand

**Comment by fernandog** [comment]
@labrys waiting your final approve & merge

**Comment by fernandog** [comment]
@labrys waiting your final approve & merge

**Comment by labrys** [comment]
Approved!

**Pull request merged**